### PR TITLE
difficulty tabbar in leaderboard page

### PR DESCRIPTION
**Pull request by changhoji**
make difficulty provider

### CODE REVIEW

File: lib/views/leaderboard/models/enums.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+enum Difficulty {
+  easy,
+  medium,
+  hard,
+}
+
+const difficulties = Difficulty.values;
```
Comment (enihsgnir) [inline lib/views/leaderboard/models/enums.dart:7]: I think there is no need to use `difficulties` instead of `Difficulty.values`.

**Review by enihsgnir — APPROVED** [review]
It looks good to me!

**Comment by changhoji** [comment]
review

**Review by enihsgnir — APPROVED** [review]
LGTM!

**Pull request merged**